api/constants/constants.go

```diff
@@ -191,6 +191,7 @@ var SystemConnectors = []string{
 }
 
 // SecondFactorType is the type of 2FA authentication.
+// Deprecated: Use types.SecondFactorType
```

Comment:
Will this cause problems on e/ ?

(I'm not sure deprecating this is wise, see other comment threads.)